project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.34")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.5")
-addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.15")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
```

Comment:
I spotted a [forum post](https://gitter.im/scalameta/scalafmt?at=5e2f15b8183449313be98b63) that seemed to suggest that the scalameta plugin used to be bad, so people used the lucid chart one, but the scalameta one is now good.